plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -45,6 +45,7 @@ public class MongoClientConfig
     private String requiredReplicaSetName;
```

Comment:
> Additionally disable local scheduling of Mongo splits

Why "additionally"

Reply:
We disable it by default as enabling couples the storage and compute layer i.e split would be assigned to the worker which is shared with MongoDB which might cause some contention of resources.